Sources/Vaporiser/MockData.swift

```diff
@@ -7,27 +7,36 @@ public struct MockData: Codable {
     public let payload: Data?
     public let method: Method
     public let returnCode: Int
+    public let body: Data?
     public let queryParameters: [URLQueryItem]?
 
-    public init(path: String, payload: Data? = nil, method: MockData.Method, queryParameters: [URLQueryItem]? = nil, returnCode: Int = 200) {
+    public init(
+        path: String,
+        payload: Data? = nil,
+        method: MockData.Method,
+        queryParameters: [URLQueryItem]? = nil,
+        body: Data? = nil,
+        returnCode: Int = 200
+    ) {
```

Comment:
I would rename payload and data with request body and response body. What do you think?